### ISSUE TITLE
Fails to compile when building from tarball instead of git clone

### ISSUE THREAD

**Issue by hauleth** [universal-ctags/ctags#727]
Way to reproduce:

```bash
$ git clone https://github.com/universal-ctags/ctags.git
$ cd ctags
$ mv .git git
$ ./autogen.sh && ./configure && make
```

This results with error:

```
In file included from main/repoinfo.c:4:0:
./main/repoinfo.h:1:1: error: expected identifier or ‘(’ before numeric constant
 0
 ^
```

**Comment by masatake** [comment]
Thank you for reporting.
Could you try #728 ?

**Comment by masatake** [comment]
Closed by #728.
